src/frontend-samples/validation-sample/walkthru.md

```diff
@@ -0,0 +1,40 @@
+# Retrieving Rules and Violations
+
+In order to retrieve the list of `validation rules`, we need to send a request to the [Validation REST API](https://developer.bentley.com/api-groups/project-delivery/apis/validation/). The Validation API allows you to create, run, and evaluate the results of validation rules. These rules are applied to each element when a test is run, and any elements found to be in violation of the rules are returned, along with the specific test that the element failed.
+
+Since the sample showcase is a read only envirornment, the sample itself does not attempt to create a test, instead using hardcoded data that matches the format of a response from the API.
```

Comment:
```suggestion
Since the sample showcase is a read-only environment, the sample itself does not attempt to create a test, instead using hardcoded data that matches the format of a response from the API.
```
typo and wording